### PR TITLE
fix(example-soap-calculator): switch to a new soap calculator webservice

### LINKED ISSUE #2444: Create self-hosted SOAP service for test
## Description / Steps to reproduce / Feature proposal
For SOAP service related tests, we rely on external soap services.  Those services might become unavailable for a short period of time or permanently.  We should create our own SOAP service running it in a container for our CI.  This work can be reused in our tests for `strong-soap` and other repos. 

### Acceptance Criteria
- [x] Create SOAP service that can be run in a docker container
- [ ] Use this container for CI in `loopback-next` repo, including updating tests.

Stretch goal:
- [ ] Use this container for CI in `strong-soap` repo.

### PR DESCRIPTION
**Pull request by raymondfeng**
Fixes #2444. 

The hosted server is a deployment of https://github.com/raymondfeng/calculator-webservice on my account@IBM Cloud using CF.

See https://calculator-webservice.mybluemix.net/calculator

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [x] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated
- [ ] Agree to the CLA (Contributor License Agreement) by [clicking and signing](https://cla.strongloop.com/agreements/strongloop/loopback-next)

### CODE REVIEW

**Comment by marioestradarosa** [comment]
@raymondfeng wouldn't be better to have this integrated in the build process and expose the service from a `spin up` container ?

**Comment by raymondfeng** [comment]
> @raymondfeng wouldn't be better to have this integrated in the build process and expose the service from a `spin up` container ?

I would love to do so but unfortunately docker is not universally available on all CI envs and developer machines. Spinning off a container also adds good amount of overhead to the build.

File: docs/site/soap-calculator-tutorial-add-controller.md [hunk 1]
```diff
@@ -106,14 +106,14 @@ service are available from here due to the dependency injection available
 through out **LB4**.
 
 ```ts
- async divide(intA: number,intB: number): Promise<DivideResult> {
+ async divide(arg0: number,arg1: number): Promise<DivideResponse> {
```
Comment (dhmlau) [inline docs/site/soap-calculator-tutorial-add-controller.md]: I personally find `intA` and `intB` a bit more descriptive than `arg0` and `arg1`.

Comment (raymondfeng) [reply docs/site/soap-calculator-tutorial-add-controller.md]: I haven't found a way to customize such argument names.

Comment (raymondfeng) [reply docs/site/soap-calculator-tutorial-add-controller.md]: Ok, I found a way :-)

**Comment by bajtos** [comment]
+1 to replace a service that's not reliable with something more robust.

@raymondfeng could you please capture somewhere in our docs that the source code behind
https://calculator-webservice.mybluemix.net/calculator can be found in 
https://github.com/raymondfeng/calculator-webservice? That way the future loopback-next maintainers will know know where to fix the service used by tests, or even setup a new hosted instance if needed.

**Pull request merged**